test/test_python_dispatch.py

```diff
@@ -1001,9 +1001,10 @@ def wrap(e):
             out.backward()
 
     def test_storage_can_be_converted_to_python_object(self):
-        with enable_torch_dispatch_mode(LoggingTensorMode):
+        # Is this test still relevant? We're just dealing with plain tensors now.
+        with enable_torch_dispatch_mode(LoggingTensor):
             s = torch.Storage()
-            z = LoggingTensorMode(torch.empty([]))
+            z = LoggingTensor(torch.empty([]))
```

Comment:
I think this is here to as a test case for a bug reported by @cbalioglu.

IIRC what we're testing here is that Storage actually works with torch_dispatch objects. So I don't think we need the mode part of it (enable_torch_dispatch_mode) and should just check that LoggingTensor(...).set_(storage) works